app/coffee/DiffManager.coffee

```diff
@@ -49,6 +50,8 @@ module.exports = DiffManager =
 			lastUpdate = updates[0]
 			if lastUpdate? and lastUpdate.v != version - 1
 				return callback new Error("latest update version, #{lastUpdate.v}, does not match doc version, #{version}")
+			
```

Comment:
Change the logic here to reflect the new situation?

If we get doc after updates, we should never have updates ahead of doc.  Make it a hard error or log it instead of discarding?

Does the check lastUpdate? and lastUpdate.v != version - 1 still apply - based on what you said about combined ops?